### ISSUE TITLE
Reorder: List not using an ordered list semantic structure (1.3.1)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#957]
### Expected behavior
List of items is in an ordered list for these semantics and list functions are available to assistive technology.

https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html

### Actual behavior
The list of items are a set of div elements.
